databases/week3/db_homework_week3.sql

```diff
@@ -0,0 +1,339 @@
+----creating the data------
+CREATE database meal_reservation_review;
+use meal_reservation_review;
+SET
+  NAMES utf8mb4;
+CREATE TABLE `meal` (
+    `id` INT(10) UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+    `title` VARCHAR(255),
+    `description` TEXT,
+    `location` VARCHAR(255),
+    `when` DATETIME,
+    `max_reservations` INT(10),
+    `price` DECIMAL,
+    `created_date` DATE
+  );
+CREATE TABLE `reservation` (
+    `id` INT(10) UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+    `number_of_guests` INT (10),
+    `meal_id` INT (10) UNSIGNED NOT NULL,
+    `created_date` DATE,
+    `contact_phonenumber` VARCHAR (25),
+    `contact_name` VARCHAR (255),
+    `contact_email` VARCHAR (255),
+    CONSTRAINT `fk_meal` FOREIGN KEY (meal_id) REFERENCES meal (id) ON DELETE CASCADE
+  );
+CREATE TABLE `review` (
+    `id` INT(10) UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+    `title` VARCHAR (255),
+    `description` TEXT,
+    `meal_id` INT(10) UNSIGNED NOT NULL,
+    `stars` INT (10),
+    `created_date` DATE,
+    CONSTRAINT `fk_meal` FOREIGN KEY (meal_id) REFERENCES meal (id) ON DELETE CASCADE
```

Comment:
Hi @OlenaKasian, You should not use the same name in multiple tables as you did in the reservation and review table when you have a foreign key. You use `CONSTRAINT `fk_meal` FOREIGN KEY (meal_id) REFERENCES meal (id) ON DELETE CASCADE`, both in review and the reservation tables. Because of the meal_id, you will not be able to create the review table. You can rename for example `fk_meal` from review to `fk_meal_review`.

Reply:
Hello @Axel, Thank you for your code review. 
I will change to `fk_meal_review`.